pages/common/license.md

```diff
@@ -2,18 +2,18 @@
 
 > Create license files for open-source projects.
 
-- Create a license:
+- Print a license to stout, using the defaults (auto-detected author name, and current year):
```

Comment:
stout => stdout :smile:  I am good with stout too by the way :stuck_out_tongue_closed_eyes: